packages/react-vapor/src/components/table-hoc/TableWithBlankSlate.tsx

```diff
@@ -21,23 +27,22 @@ export const tableWithBlankSlate = (supplier: ConfigSupplier<IBlankSlateWithTabl
     const config = HocUtils.supplyConfig(supplier);
     const defaultRenderBlankSlateMethod = <BlankSlateWithTable {...config} />;
     const mapStateToProps = (state: IReactVaporState, ownProps: ITableHOCOwnProps & ITableWithBlankSlateProps) => {
-        const isEmpty = ownProps.renderBlankSlateOnly
-            ? TableSelectors.getIsTruelyEmpty(state, ownProps)
-            : TableSelectors.getIsEmpty(state, ownProps);
+        const isEmpty = TableSelectors.getIsEmpty(state, ownProps);
         return {
             isEmpty,
+            isTruelyEmpty: TableSelectors.getIsTruelyEmpty(state, ownProps),

```

Comment:
The next prop will be `isIrremediablyForeverEmpty`

Reply:
Is istruelyEmpty truly empty? :thinking: :philosophe:

Oh btw -->  truly, not truely :sweat_smile:

Reply:
Good catch 👍  I can only change it for the prop though, because changing the selector would be a breaking change 😓

Reply:
:cry:

Reply:
I renamed everything I could and deprecated what I couldn't